server/pbench/bin/pbench-verify-s3-backup-tarballs

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import glob
+import hashlib
+import boto3
+import configtools
+
+options = []
+opts, args = configtools.parse_args(options)
+conf, files = configtools.init(opts)
+
+# configuration file
+endpoint_url = conf.get('aws-s3', 'endpoint_url')
+aws_access_key_id = conf.get('aws-s3', 'aws_access_key_id')
+aws_secret_access_key = conf.get('aws-s3', 'aws_secret_access_key')
+bucket_name = conf.get('aws-s3', 'bucket_name')
+target_dir = conf.get('aws-s3', 'target_dir')
+
+s3 = boto3.resource('s3', aws_access_key_id='%s' % aws_access_key_id,
+                    aws_secret_access_key='%s' % aws_secret_access_key, endpoint_url='%s' % endpoint_url)
+
+def checkmd5(target_dir, list_dir, indicator):
+    tarlist = glob.glob('%s/*/*.tar.xz' % target_dir)
+    for tar in tarlist:
+        controller = tar.split("/")[-2]
+        result_name = tar.split("/")[-1]
+        md5 = ("%s.md5" % tar)
+        with open("%s/list.%s" % (list_dir, indicator), 'a') as f:
```

Comment:
Are we sure we want to "append" to an existing `list.*` file?  Or do we want to use the "`w`" mode when opening a file so that we create it as a new list with only the contents we find in the `tarlist`?

Reply:
Done